client/src/controllers/TagController.ts

```diff
@@ -20,17 +21,31 @@ export class TagController extends Controller {
   static values = {
     options: { default: {}, type: Object },
     url: String,
+    debounce: { type: Number, default: 400 },
```

Comment:
Let's default this to 300 as you have in your comment below

```suggestion
    debounce: { type: Number, default: 300 },
```

Reply:
Alright, whatever feels better